src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -417,6 +417,10 @@ class Wav2Vec2Config:
     max_spatial_mask_prob: float
     """The maximum probability of masking a feature. Note that, due to mask span
     overlap, the effective probability might be smaller."""
+    
+    min_num_spans: int
+    """The minimum number of mask sampled per sequences. Note that it is suggested
```

Comment:
` Note that it is suggested ...` where ?

Reply:
Suggested as in it was the previous value. I can remove the note, I agree that it is confusing!

Reply:
Yeah lets remove this :)